src/constants.js

```diff
@@ -381,6 +381,7 @@ export const VERSION_RETENTION_INDEFINITE: 'indefinite' = 'indefinite';
 
 /* ------------------ Placeholder Feed Items ------------------------- */
 export const PLACEHOLDER_USER = { type: 'user', id: '2', name: '' };
+export const FILE_REQUEST = 'File Request';
```

Comment:
Is this meant to be equivalent a user name? Can we call it `FILE_REQUEST_NAME` to indicate that?